test.js

```diff
@@ -383,4 +383,407 @@ test("isValidMove", t => {
             );
         });
     });           
-});
\ No newline at end of file
+});
+
+const makeMove = require("./").makeMove;
+
+test("makeMove", t => {
+    
+    t.test("moving to an empty space", t => {
+        const moveToEmptySpace = {
+            currentPosition: 10,
+            numberOfSpaces: 3,
+            isBar: false
+        };
+        
+        t.test("player 1", t => { 
+            t.test("moving from space with 1 piece", t => {
+                t.plan(1);
+                const oldGameState = getInitialGameState();
+                oldGameState.board[10].isPlayerOne = true;
+                oldGameState.board[10].numPieces = 1;
+                
+                const newGameState = getInitialGameState();
+                newGameState.board[13].isPlayerOne = true;
+                newGameState.board[13].numPieces = 1;
+                newGameState.isPlayerOne = false;
+                
+                t.deepEqual(
+                    makeMove(oldGameState, moveToEmptySpace),
+                    newGameState,
+                    "the correct game state is returned when player 1 moves from a space with 1 piece to an empty space"
+                );
+            });          
+            
+            t.test("moving from space with 2 pieces", t => {
+                t.plan(1);
+                const oldGameState = getInitialGameState();
+                oldGameState.board[10].isPlayerOne = true;
+                oldGameState.board[10].numPieces = 2;
+                
+                const newGameState = getInitialGameState();
+                newGameState.board[10].isPlayerOne = true;
+                newGameState.board[10].numPieces = 1;
+                newGameState.board[13].isPlayerOne = true;
+                newGameState.board[13].numPieces = 1;
+                newGameState.isPlayerOne = false;
+                
+                t.deepEqual(
+                    makeMove(oldGameState, moveToEmptySpace),
+                    newGameState,
+                    "the correct game state is returned when player 1 moves" + 
+                    "from a space with 2 pieces to an empty space"
```

Comment:
You should indent this another level so it's obvious that it's a continuation of the string on the previous line and not a fourth argument to the function:

```js
t.deepEqual(
    a,
    b,
    'message ' +
        'continuation'
);
```